pkg/models/message.go

```diff
@@ -53,11 +54,12 @@ func (m *Message) GetError() error {
 
 func (m *Message) String() string {
 	return fmt.Sprintf(
-		"PartitionKey:%s,TimeCreated:%v,TimePulled:%v,TimeTransformed:%v,Data:%s",
+		"PartitionKey:%s,TimeCreated:%v,TimePulled:%v,TimeTransformed:%v,HttpHeaders:%v,Data:%s",
```

Comment:
In a batch model this doesn't make sense. I wouldn't add it to the logging here.

Reply:
Ok, makes sense. I'll remove it.